cmd/create.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	candidateRegistry string = ""
+	containerName     string = ""
+	imageName         string = ""
+	releaseVer        int    = 30
+)
+
+// createCmd represents the create command
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a new toolbox container",
+	Run: func(cmd *cobra.Command, args []string) {
+		create()
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(createCmd)
+
+	// Here you will define your flags and configuration settings.
+
+	// Cobra supports Persistent Flags which will work for this command
+	// and all subcommands, e.g.:
+	// createCmd.PersistentFlags().String("foo", "", "A help for foo")
+
+	// Cobra supports local flags which will only run when this command
+	// is called directly, e.g.:
+	createCmd.Flags().StringVarP(&candidateRegistry, "canidate-registry", "", false, "Pull the base image from candidate-regostry.fedoraproject.org")
```

Comment:
Let's not copy this one from shell-toolbox; I think we should just support pulling any base image and not special case "the" toolbox image too much.

Reply:
would a config file with a default be useful?

Reply:
Ok, I was just mirroring the options of containers/toolbox.

Reply:
To rephrase here:

Right - part of my point here is I don't think we should think of this as "rewriting original shell toolbox in Go" - it's more like "merge our toolbox efforts".  Using original shell toolbox for inspiration of course makes total sense, but I'd like to *also* use as a reference the code in https://github.com/containers/toolbox/pull/319

Reply:
That's my intention. I'm still getting to know Go and Cobra so I took the thing I knew the most and started to experiment.